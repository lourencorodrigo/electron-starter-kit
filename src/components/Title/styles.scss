.title {
  font-size: 1.5em;
  color: #848484;
  margin: 0;
}