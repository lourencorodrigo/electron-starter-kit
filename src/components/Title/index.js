import React from 'react';

import styles from './styles.scss';

const Title = () => (
  <h1 className={styles.title}>Welcome to React</h1>
);

export default Title;
