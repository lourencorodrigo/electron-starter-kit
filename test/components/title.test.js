import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import { mount } from 'enzyme';

import Title from '../../src/components/Title';

describe('TITLE COMPONENT', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = mount(<Title />);
  });

  it('+++ should compare the text in h1.', () => {
    expect(wrapper.find('h1').render().text()).toEqual('Welcome to React');
  });

  it('+++ should compare the component with a snapshot', () => {
    const component = ReactTestRenderer.create(<Title />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });

});
