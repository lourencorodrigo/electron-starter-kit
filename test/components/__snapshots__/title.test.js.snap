// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TITLE COMPONENT +++ should compare the component with a snapshot 1`] = `
<h1
  className="title"
>
  Welcome to React
</h1>
`;
